### PR TITLE
Submission of Week-01 Assignments

### PR DESCRIPTION
**Pull request by GopalDev22**
Hi Suraj,

Submitting Week-01 Assignments.

Please review and merge the code.

Thanks,
Venu Gopal.

### CODE REVIEW

File: Week-1/Session-02-HTML_Deployment/Submission/Intermediate/Exercise-01/index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Hometown Webpage</title>
+  </head>
+  <body>
+    <h1>sometown, USA</h1>
```
Comment (suraj7091) [inline Week-1/Session-02-HTML_Deployment/Submission/Intermediate/Exercise-01/index.html:10]: not containing section tags as described in point 3

File: Week-1/Session-01/submission/assignment.html
Comment (suraj7091) [inline Week-1/Session-01/submission/assignment.html:1]: The submission should be in .md file for this

File: Week-1/Session-02-HTML_Deployment/Submission/Intermediate/Exercise-03/poem.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      Then there's pair of us! <br />
+      Don't tell! They'd banish us- you know! <br />
+    </p>
+    <p>
+      How dreary - to be - Somebody! <br />
+      How public - like a Frog - <br />
+      To tell one's name - the livelong June - <br />
+      To an admiring Bog! <br />
+    </p>
+    <cite>By Emily Dickinson (1891)</cite>
```
Comment (suraj7091) [inline Week-1/Session-02-HTML_Deployment/Submission/Intermediate/Exercise-03/poem.html]: use third paragraph for this

File: Week-1/Session-02-HTML_Deployment/Submission/Advanced/newArticle.html
Comment (suraj7091) [inline Week-1/Session-02-HTML_Deployment/Submission/Advanced/newArticle.html:1]: Article photo as  tag with src attribute containing SnakeBite.jpg and alt
attribute containing Snake bite on side of child's foot missing

**Review by suraj7091 — CHANGES_REQUESTED** [review]

File: Week-1/Session-02-HTML_Deployment/Submission/Advanced/newArticle.html
Comment (GopalDev22) [reply Week-1/Session-02-HTML_Deployment/Submission/Advanced/newArticle.html:1]: updated as per the suggestion.

File: Week-1/Session-02-HTML_Deployment/Submission/Intermediate/Exercise-03/poem.html (hunk 2 above)
Comment (GopalDev22) [reply Week-1/Session-02-HTML_Deployment/Submission/Intermediate/Exercise-03/poem.html]: updated as per suggestion

File: Week-1/Session-02-HTML_Deployment/Submission/Intermediate/Exercise-01/index.html (hunk 1 above)
Comment (GopalDev22) [reply Week-1/Session-02-HTML_Deployment/Submission/Intermediate/Exercise-01/index.html:10]: updated as per suggestion.

File: Week-1/Session-01/submission/assignment.html
Comment (GopalDev22) [reply Week-1/Session-01/submission/assignment.html:1]: created assignment in .md files. please review.

**Review by suraj7091 — APPROVED** [review]

**Pull request merged**